homework3.ts

```diff
@@ -0,0 +1,40 @@
+function required(message: string) {
+    return (target: object, PropertyKey: string) => {
+        let value: string;
+        const getter = function() {
+            return value;
+        };
+  
+        const setter = function(nam: string) {
+            if (nam.length !== 0) {
+                value = nam;
+            } else {
+                value = message;
```

Comment:
You shouldn't assign message to to name of the person. Just log error message here.